### DISCUSSION TITLE
CI pipeline step "Setup ruby" exits with exit code 5

### DISCUSSION THREAD

**Discussion by mhelmerich** [cotes2020/jekyll-theme-chirpy#1428] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Hey guys,

I tried to push my first chirpy doc site but I'm running into an issue here.
In the Setup ruby step I'm getting the following error message:

```
An error occurred while installing google-protobuf (3.25.1), and Bundler cannot
continue.

In Gemfile:
  jekyll-theme-chirpy was resolved to 6.3.1, which depends on
    jekyll-archives was resolved to 2.2.1, which depends on
      jekyll was resolved to 4.3.2, which depends on
        jekyll-sass-converter was resolved to 3.0.0, which depends on
          sass-embedded was resolved to 1.69.5, which depends on
            google-protobuf
Error: The process '/opt/hostedtoolcache/Ruby/3.3.0/x64/bin/bundle' failed with exit code 5
```

### Operations you have already tried

I tried rerunning the CI pineline several times but it always exits at the same step.

### Anything else?

The markdown file I tried to publish can be found [here](https://github.com/mhelmerich/mhelmerich.github.io/blob/main/_posts/2023-12-25-jekyll.md), from my point of view I did everything according to the documentation.

**Comment by cotes2020** [comment, marked as answer]
Thanks for reporting this issue, we've since discussed it in:
- #1429.

**Reply by mhelmerich** [reply]
Thank you, got it working!

**Comment by xiaoyouPrince** [comment]
i have the same issue,  from 2 days ago, i'm. looking for the result,  just now, i found it has success. may be the network?
